.github/workflows/rust-fmt.yml

```
@@ -0,0 +1,31 @@
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+          components: rustfmt
+      
+      - name: Check formatting
+        uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)